usaspending_api/awards/models.py

```diff
@@ -102,9 +103,9 @@ class Award(DataSourceTrackedModel):
         ('L', 'Loan'),
     )
 
-    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type")
+    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type", null=True)
```

Comment:
Sure, this mainly because when creating a parent summary award we can't know the type of that vehicle, so I leave it null.